Cargo.toml

```diff
@@ -27,3 +27,7 @@ path = "src/lib.rs"
 
 [[bin]]
 name = "parsec-tool"
+
+[patch.crates-io]
+parsec-client = { git = "https://github.com/joechrisellis/parsec-client-rust.git", branch = "add-list-keys" }
+parsec-interface = { git = "https://github.com/joechrisellis/parsec-interface-rs.git", branch = "add-list-keys" }
```

Comment:
You can directly modify the `parsec-client` and `parsec-interface` dependencies to:

```toml
[dependencies]
parsec-client = { git = "https://github.com/joechrisellis/parsec-client-rust.git", branch = "add-list-keys" }
parsec-interface = { git = "https://github.com/parallaxsecond/parsec-interface-rs.git" }
```

Depending if we merge your PR of `parsec-client` on `master`, you will be able to use directly the `origin/master` version.

Reply:
Also you should only need `parsec-client` here I think since you get the interface from it as a public module.